hivemind/proto/p2pd.proto

```diff
@@ -7,18 +7,23 @@ syntax = "proto2";
 package p2pclient.p2pd.pb;
 
 message Request {
-  enum Type {
-    IDENTIFY       = 0;
-    CONNECT        = 1;
-    STREAM_OPEN    = 2;
-    STREAM_HANDLER = 3;
-    DHT            = 4;
-    LIST_PEERS     = 5;
-    CONNMANAGER    = 6;
-    DISCONNECT     = 7;
-    PUBSUB         = 8;
-  }
+   enum Type {
+    IDENTIFY                 = 0;
+    CONNECT                  = 1;
+    STREAM_OPEN              = 2;
+    STREAM_HANDLER           = 3;
+    DHT                      = 4;
+    LIST_PEERS               = 5;
+    CONNMANAGER              = 6;
+    DISCONNECT               = 7;      
+    PUBSUB                   = 8;
+
+    PERSISTENT_CONN_UPGRADE  = 9;
+    CALL_UNARY              = 10;
+    ADD_UNARY_HANDLER       = 11;
+    SEND_RESPONSE_TO_REMOTE = 12;
```

Comment:
```suggestion
    RETURN_UNARY_RESPONSE = 12;
```

This can be fixed after running the DHT benchmark :)

Reply:
Enums from 10 to 12 are actually deprecated -- I'll remove them later.